src/licensedcode/models.py

```diff
@@ -479,6 +481,8 @@ def dump(self, licenses_data_dir):
         """
         metadata = self.to_dict(include_builtin=False)
         content = self.text
+        # We add a space to empty newlines to make this text YAML safe
+        content = get_yaml_safe_text(content)
```

Comment:
What about rules? there must be a good bunch of these that are also incorrect?

Reply:
For some reason rules don't fail. This could be from a whitespace stripping operation applied to the left of the text previously (the rules don't seem to have first line indented)